egs2/how2_2000h/asr2/run.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+
+kmeans_feature="wavlm_large/21"  # use model_type/layer_index
```

Comment:
We can temporarily use wavlm_large. But it need to be changed later to use fbank_pitch.